condition_logic.go

```diff
@@ -0,0 +1,98 @@
+package growthbook
```

Comment:
Agree, I thought to split project into packages later, but it worth to do it from the start it seems. As condition are not supposed to be used by end users, it could became internal package I think.  

 This is not finished yet, so testing will be added. GrowthBook also has common test suite (in form of JSON file) that used by all SDK implementations.

Do you mean separate by using different files in the same gbcond package or to move deseralization somewhere else?